### ISSUE TITLE
Paginator: page html input use number type attribute

### ISSUE THREAD

**Issue by djmj** [primefaces/primefaces#14944]
### Description

For easier accessibility, especially on mobile devices keyboard it would be nice if paginator page select input field would have following attributes:

`type="number" min="1" max="10" step="1"`

Maybe `size` is als a good idea calculated based on the string length of 'max' value. Then it would not default to a large input field if only 9 pages are present. But in my experience `size` is tricky depending on browser or if spinner buttons are rendered like in chrome.

### Describe the solution you would like

Paginator page select input html should be like:

`<input class="ui-paginator-jtp-input ui-inputfield ui-inputtext ui-widget ui-state-default ui-state-filled" value="1" type="number" min="1" max="10" step="1" aria-label="Zum Eingabefeld springen">`

**Comment by djmj** [comment]
Much thanks!

**Comment by melloware** [comment]
you can see it in action here: https://primefaces-showcase-zvbjj.ondigitalocean.app/ui/data/datatable/paginator
